README.md

```diff
@@ -100,11 +113,22 @@ export CHPL_RT_OVERSUBSCRIBED=yes
 cd $CHPL_HOME
 make
 ```
+# If you've built Chapel instead using brew, add the Chapel executable (chpl) to PATH:
+export PATH=$CHPL_HOME/bin/linux64-x86_64/:$PATH
+```
```

Comment:
This and the above anaconda install look linux specific, but are listed under the macos instructions. The header formatting with `###` also seems to have thrown some other formatting off -- https://github.com/mhmerrill/arkouda/blob/55f8a612664417c1d3795cded37079b47bb88538/README.md